web/src/components/MapView.tsx

```diff
@@ -36,6 +50,7 @@ type MapViewProps = {
   children: ReactNode
   viewport?: MapViewViewport
   setViewport: (mapViewport: MapViewViewport) => void
+  Overlay?: ReactElement
```

Comment:
this shouldn't be optional?

Reply:
I disagree. If we ever want to reuse the map view it should not be required to render an overlay.